tap_pendo/streams.py

```diff
@@ -406,7 +406,9 @@ def sync_substream(self, state, parent, sub_stream, parent_response):
                             integer_datetime_fmt=
                             "unix-milliseconds-integer-datetime-parsing"
                         ) as transformer:
-                    stream_events = sub_stream.sync(state, new_bookmark,
+                    # bug fix for syncing child streams from start date or
+                    # state file date and not newly updated bookmark
+                    stream_events = sub_stream.sync(state, bookmark_dttm,
```

Comment:
@savan-chovatiya Can you please add unit test cases for this code change

Reply:
Added unittest.